core/src/avm2/globals/flash/events/GestureEvent.as

```diff
@@ -0,0 +1,50 @@
+package flash.events
+{
+    
+    public class GestureEvent extends Event
```

Comment:
I think `updateAfterEvent()` is missing here. It is also not present in `MouseEvent` ... but why could that be?

Reply:
I saw that it was missing in `MouseEvent` and replicated that behavior, maybe at the time (even now?) the API isn't complete enough to be able to do it?

Reply:
Yeah, that's true, it wouldn't work anyway, and it's just not stubbed either.